xml/System.Windows.Forms/InputLanguage.xml

```diff
@@ -237,23 +237,21 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Windows.Forms.InputLanguage.FromCulture%2A> method selects from among installed keyboards. Specifically, this method performs the following operations:  
+The <xref:System.Windows.Forms.InputLanguage.FromCulture%2A> method returns the first keyboard layout with matching `culture` from the list of all installed keyboard layouts and therefore may return an unexpected value if multiple keyboards are installed under a single user language.
```

Comment:
```suggestion

The <xref:System.Windows.Forms.InputLanguage.FromCulture%2A> method returns the first keyboard layout with matching `culture` from the list of all installed keyboard layouts. It might return an unexpected value if multiple keyboards are installed under a single user language.
```